### ISSUE TITLE
LinkDataCollection inside a property list causes property list to stop working

### ISSUE THREAD

**Issue by victortranstockland** [Geta/geta-optimizely-genericlinks#24]
When there is a LinkDataCollection inside a property list, the property list stops responding correctly - images cannot be dragged from the media gallery and updates stop being saved to the database.

Optimizely CMS 12.19.0
Optimizely CMS UI 12.23.0

**Comment by GeekInTheNorth** [comment]
@victortranstockland I'm having a similar issue on the latest version of the Optimizely CMS  where actually the Content Item does not fully load in the CMS Editor Interface.

```csharp
public class NavigationConfigurationPage : PageData
{
// These properties load
    public virtual string? MainNavigationPrimaryTitle { get; set; }

    public virtual ContentReference? MainNavigationPrimaryImage { get; set; }

// This property does not load
    public virtual LinkDataCollection<LinkItemWithImage>? MainNavigationPrimaryLinks { get; set; }

// Subsequent properties also do not appear to load after the failure on the above property.
    public virtual string? MainNavigationSecondaryTitle { get; set; }

    public virtual ContentReference? MainNavigationSecondaryImage { get; set; }

    public virtual LinkDataCollection<LinkItemWithImage>? MainNavigationSecondaryLinks { get; set; }
}
```

**Comment by svenrog** [comment]
I couldn't reproduce this.
Although I've fixed a related issue in #25 with version 1.8.1, you can try testing that. It might be the same issue.

**Comment by svenrog** [comment]
Nuget found [here](https://nuget.optimizely.com/package/?id=Geta.Optimizely.GenericLinks).

**Comment by GeekInTheNorth** [comment]
Hello @svenrog I'll see if I can reproduce the issue with the fix in place and get back to you

**Comment by victortranstockland** [comment]
@svenrog Looks like my issue was resolved in 1.8.1.
